engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EmailTemplateRepository.java

```diff
@@ -15,15 +15,15 @@
 import static com.redhat.cloud.notifications.models.EmailSubscriptionType.INSTANT;
 
 @ApplicationScoped
-public class TemplateRepository {
+public class EmailTemplateRepository {
 
     @Inject
     StatelessSessionFactory statelessSessionFactory;
 
     public boolean isEmailSubscriptionSupported(String bundleName, String appName, EmailSubscriptionType subscriptionType) {
         if (subscriptionType == INSTANT) {
-            String hql = "SELECT COUNT(*) FROM InstantEmailTemplate " +
-                    "WHERE eventType.application.bundle.name = :bundleName AND eventType.application.name = :appName";
+            String hql = "SELECT COUNT(*) FROM InstantEmailTemplate WHERE enabled IS TRUE " +
```

Comment:
Should we allow subscription for `instant` notifications when the event type is linked with a disabled template?

Reply:
I don't think we should allow that.